### PR TITLE
More widgets: insert TitleBar

### PR DESCRIPTION
**Pull request by hius07**
TitleBar: add info_text
SpinWidget, DoubleSpinWidget, DateTimeWidget, RadioButtonWidget: insert TitleBar

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/8683)

### CODE REVIEW

File: frontend/ui/widget/spinwidget.lua [hunk 1]
```diff
@@ -111,21 +109,15 @@ function SpinWidget:update(numberpicker_value, numberpicker_value_index)
...
-        dimen = Geom:new{
-            w = self.width,
-            h = Size.line.thick,
-        }
+    local title_bar = TitleBar:new{
+        width = self.width,
+        align = "left",
+        with_bottom_line = true,
+        title = self.title_text,
+        title_face = self.title_face,
```
Comment (poire-z) [inline frontend/ui/widget/spinwidget.lua:117]: Does it make sense to keep allowing these widgets' title_face to be customizable ? Do we have some instances using a different font ? Shouldn't we prevent that to enforce consistency ?

**Comment by poire-z** [comment]
Not sure about my feeling, so asking: doesn't the padding between the title bottom line and the info_text feel quite small ? It feels a bit constrained to me (vs. the one we have at the bottom of the main content), but I dunno, may be that's what we had before :)
image

**Comment by hius07** [comment]
New - old

**Comment by hius07** [comment]
Without bottom line no need to add more padding.



With bottom line it can be default or large (I think large is good)

**Comment by hius07** [comment]
Looking at both SW and DSW, default padding is better (it is in the left picture)

**Pull request merged**